test/test_main.cpp

```diff
@@ -396,7 +396,7 @@ void RelatedGL(const Manifold& out, const std::vector<MeshGL>& originals,
             const float volumeP =
                 glm::dot(edgesP[0], glm::cross(edgesP[1], edgesP[2]));
 
-            ASSERT_LE(volumeP, area * 100 * out.Precision());
+            ASSERT_LE(volumeP, area * out.Precision());
```

Comment:
Our regular tests would have caught this bug if I hadn't put in this *100, which has the look of me just getting frustrated at some point.

Reply:
interesting, I think we should probably go over all tests to see if there are suspicious ones